spec/baked_file_system_spec.cr

```diff
@@ -13,13 +13,13 @@ describe BakedFileSystem do
     baked_file = Storage.get("images/sidekiq.png")
     baked_file.name.should eq("sidekiq.png")
     baked_file.size.should eq(52949)
-    baked_file.compressed_size.should eq(47883)
+    baked_file.compressed_size.should be_close 47883, 40
```

Comment:
There are slight derivations of the compressed size between calling `gzip` and using `Gzip::Writer` and the exact values don't matter. Though it could probably replaced with the values produced by this PR.